README.md

```diff
@@ -11,15 +11,13 @@ You can download and use WinUI packages in your app using the NuGet package mana
 
 ### Packages
 
-| NuGet Package Name | Description |
-| --- | --- |
-| [Microsoft.UI.Xaml](https://www.nuget.org/packages/Microsoft.UI.Xaml) | Controls and features for building apps for Windows 10 |
-| [Microsoft.UI.Xaml.Core.Direct](https://www.nuget.org/packages/Microsoft.UI.Xaml.Core.Direct) | Low-level APIs for creating middleware components. Not intended or required for general application use |
+| NuGet Package | Build Status | Latest Versions |
+| --- | --- | --- |
+| [Microsoft.UI.Xaml](https://www.nuget.org/packages/Microsoft.UI.Xaml) <br /> Controls and Fluent Design for UWP apps | [![Build Status](https://dev.azure.com/ms/microsoft-ui-xaml/_apis/build/status/WinUI-Public-MUX-CI?branchName=master)](https://dev.azure.com/ms/microsoft-ui-xaml/_build/latest?definitionId=20?branchName=master) | ![latest stable version](https://img.shields.io/nuget/v/Microsoft.UI.Xaml.svg) <br /> ![latest prerelease version](https://img.shields.io/nuget/vpre/Microsoft.UI.Xaml.svg) |
```

Comment:
Can the nuget badges link out to nuget.org? I'm not sure if there's a way to get a link to "latest" or what the convention here is.